File: evals/fixtures/fastapi-app/src/services/payment.py
import stripe
from src.core.config import settings

async def charge(order):
    try:
        intent = stripe.PaymentIntent.create(amount=order["total"])
        return {"ok": True, "intent_id": intent.id}
    except Exception as e:
        return {"ok": False, "reason": str(e)}

File: evals/fixtures/express-app/src/services/payment.ts
import Stripe from 'stripe'
import { config } from '../config/env'

const stripe = new Stripe(config.stripeKey)

export async function chargePayment(order: Order): Promise<ChargeResult> {
  try {
    const intent = await stripe.paymentIntents.create({ amount: order.total })
    return { ok: true, intentId: intent.id }
  } catch (e) {
    return { ok: false, reason: e.code }
  }
}
